src/theme/ThemeProvider.tsx

```diff
@@ -1,4 +1,6 @@
+import type { Theme } from "@material-ui/core/styles";
 import { MuiThemeProvider } from "@material-ui/core/styles";
+import { merge } from "lodash";
```

Comment:
Isn't it better to change it to?:
```suggestion
import merge from "lodash/merge";
```

Reply:
`tsdx` has babel plugin for this, so no need to use deep imports